Cannot "make"

Hi, when I run "make" or "make install" in plugins/outotune/, after installing the dependencies and running both git submodule commands, I get the following as output:

Creating LV2 plugin library for outotune
/usr/bin/ld: cannot find -lworld: No such file or directory
collect2: error: ld returned 1 exit status
make: *** [../../dpf/Makefile.plugins.mk:218: ../../bin/outotune.lv2/outotune_dsp.so] Error 1

Running "locate libworld" doesn't return anything, if that's of any help.

I'm not sure if it's trying to access outotune/World or not, but either way I don't really know how to solve it. If this is an error on my end, please let me know and I'll remove the issue, but as of now I don't really know how to proceed. Thanks for your work!

--- a/plugins/outotune/Makefile
+++ b/plugins/outotune/Makefile
@@ -61,10 +61,10 @@ BUILD_CXX_FLAGS += -std=c++17 -L../../World/build -lworld -I ../../World/src -g 
 
 EXTERNAL_DEPS = ../../World/build/libworld.a
 
-jack: $(EXTERNAL_DEPS)
-lv2_dsp: $(EXTERNAL_DEPS)
-lv2_sep: $(EXTERNAL_DEPS)
-vst: $(EXTERNAL_DEPS)
+../../bin/outotune: $(EXTERNAL_DEPS)
+../../bin/outotune.lv2/outotune_dsp.so: $(EXTERNAL_DEPS)
+../../bin/outotune.lv2/outotune_ui.so: $(EXTERNAL_DEPS)
+../../bin/outotune-vst.so: $(EXTERNAL_DEPS)
 
 all: $(TARGETS) ../../bin/outotune.lv2/manifest.ttl
 

